src/components/navBar.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Navbar() {
+  return (
+    <nav className="navbar">
+      <Link to="/">CONVENE</Link>
```

Comment:
The <NavLink> tag matches better for this component since it's part of the site navigation